mcprag/mcp/tools/azure_management.py

```
@@ -218,7 +219,7 @@ async def manage_documents(
             return err("Data automation not available")
 
         from ...config import Config
```

Ruff (0.12.2)

`...config.Config` imported but unused

Remove unused import: `...config.Config`

(F401)